### DISCUSSION TITLE
Intel ARC Support

### DISCUSSION THREAD

**Discussion by Noble772** [Purfview/whisper-standalone-win#262] [Q&A] [answered]
I can't seem to figure out how to get xpu support or if it is even possible. Has anyone figured out a way to use their Intel Arc GPUs?

**Comment by Purfview** [comment, marked as answer]
Only GPUs with CUDA are supported.

**Comment by Noble772** [comment]
Yep I figured that out on another post you made on SubtitleEdit's repo. Looks like I'm putting my 1050ti back in with my Arc GPU.
